Makefile

```diff
@@ -14,6 +14,7 @@ test:
 	fi
 	
 	PYTHONPATH=.:$(KIWI_INCLUDE_PATH) EXECUTOR=standard PYTHONWARNINGS=d AUTO_CREATE_SCHEMA='' \
+        KIWI_TENANTS_DOMAIN='test.com' \
 	    coverage run --include "tcms_tenants/*.py" \
```

Comment:
This change is needed b/c of the fact that checks are executed automatically upon migrate (which is triggered by test), correct ? 

We'll also need an additional test, just to make sure the registration portion is working. I think the easiest way is probably to duplicate this line and call `manage.pt check`, not set the ENV variable and grep for the expected error message.